route/build/src/read_control.f90

```diff
@@ -342,6 +344,13 @@ SUBROUTINE read_control(ctl_fname, err, message)
    restart_dir = output_dir
  endif
 
+ ! ---------- restart option  ---------------------------------------------------------------------
```

Comment:
Make sure this only happens for standalone.

Reply:
Or it could be done, but we ignore it for the coupled case?

Reply:
This can be done by setting runMode earlier in rof_comp_nuopc, so it can be accessed at this point.

Reply:
Added if statement to set isRestart only if `runMode` is standalone